.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
+    "ms-azuretools.vscode-azurefunctions",
+    "vscjava.vscode-java-debug"
```

Comment:
Are these required?

Reply:
Same question for other files added to .vscode directory.

Reply:
This was created when I added the java azure function from CLI. Not required I guess, will remove them.